src/test/scala/intent/runner/FocusedTest.scala

```diff
@@ -51,7 +51,7 @@ class FocusedTest extends TestSuite with State[FocusedTestCase] with
   )
 
 case class FocusedTestCase(
-  suiteClassName: String = null,
+  suiteClassName: String | Null = null,
```

Comment:
**Nullable variable**
Variables that really can be null and should therefore have a `| Null` type.